test/libyul/YulOptimizerTest.cpp

```diff
@@ -349,7 +349,7 @@ TestCase::TestResult YulOptimizerTest::run(ostream& _stream, string const& _line
 	{
 		disambiguate();
 		ExpressionSplitter::run(*m_context, *m_ast);
-		WordSizeTransform::run(*m_dialect, *m_ast, *m_nameDispenser);
+		WordSizeTransform::run(*m_dialect, ""_yulstring, *m_ast, *m_nameDispenser);
```

Comment:
What happens with empty default type?

Reply:
This is the type that is used for the split variables. If it's empty then it's empty. Maybe I don't understand the question.